MeesGame/Gameplay/Generator/MazeGeneratorInit.cs

```diff
@@ -154,7 +154,7 @@ private void InitialiseExitSearch()
         {
             // The exit will be placed at the hardest to reach tile, so keep an array for that.
             exitScore = new int[numRows, numCols];
-            bestExit = start;
+            bestExit = new Point(0, 1);
```

Comment:
Why this change? There's an error handler in case the best exit remains the start and the start isn't on the edge. Using magic values always comes with a risk.